src/annotator/guest.js

```diff
@@ -484,7 +478,7 @@ export default class Guest extends Delegator {
         ignoreSelector: IGNORE_SELECTOR,
       };
       // Returns an array of selectors for the passed range.
-      return this.anchoring.describe(root, range, options);
+      return htmlDescribe(root, range, options);
```

Comment:
As in the `anchor` method, `this.anchoring` needs to be used to call the appropriate implementation depending on the document type (HTML vs PDF)

Reply:
awe, that was not so obvious. I might add a comment too